recipes/configs/llama3_1/70B_full.yaml

```diff
@@ -104,6 +104,6 @@ dtype: bf16
 metric_logger:
   _component_: torchtune.utils.metric_logging.DiskLogger
   log_dir: ${output_dir}
-output_dir: /tmp/alpaca-llama3-finetune
+output_dir: /tmp/alpaca-llama3_1-finetune
```

Comment:
mind changing this to `/tmp/full-llama3_1-finetune-output` or `/tmp/llama3_1-finetune-output` while you're here?